pkg/spireentry/entries.go

```diff
@@ -56,12 +56,16 @@ func renderPodEntry(spec *spirev1alpha1.ParsedClusterSPIFFEIDSpec, node *corev1.
 	}
 
 	var dnsNames []string
+	dnsNamesUnique := make(map[string]struct{})
```

Comment:
Not happy with this name for the map, but couldn't think of anything better.

Reply:
How about `dnsNamesSet`?